### ISSUE TITLE
[nexus]Please add support for installing plugins

### ISSUE THREAD

**Issue by koushyk** [Oteemo/charts#283]
It would be nice to have possibility install plugins or make install dir persistent.

**Comment by msaperst** [comment]
yes. not being able to set multiple persistent volumes, or the install dir persistent, or install plugins, makes installing plugins very difficult
